packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -65,3 +70,22 @@
 .edit-site-screen-background-color__control {
 	padding: $grid-unit-20;
 }
+
+.edit-site-global-styles-variations_item {
+	box-sizing: border-box;
+	padding: 3px;
```

Comment:
Why is padding 3px ? would it make sense to use $block-selected-margin or $grid-unit-05 (4px) - $border-width (1px)?

Reply:
It's kind of random, it's just the white zone between the border and the actual preview. I can probably use $grid-unit-05